docs/tutorial/shortcuts.md

```diff
@@ -0,0 +1,94 @@
+# Keyboard Shortcuts
+
+> Configure local and global keyboard shortcuts
+
+## Local Shortcuts
+
+You can use the [Menu] module to configure keyboard shortcuts that will
+be triggered only when the app is focused. To do so, specify an
+[`accelerator`] property when creating a [MenuItem].
+
+```js
+const {Menu, MenuItem} = require('electron')
+const menu = new Menu()
+
+menu.append(new MenuItem({
+  label: 'Print',
+  accelerator: 'CmdOrCtrl+P'
+  click: () => { console.log('time to print stuff') }
+}))
+```
+
+It's easy to configure different key combinations based on the user's operating system.
+
+```js
+{
+  accelerator: process.platform === 'darwin' ? 'Alt+Cmd+I' : 'Ctrl+Shift+I'
+}
+```
+
+If you want to configure a local keyboard shortcut to trigger an action that
+_does not_ have a corresponding menu item, you can use the
+[electron-localshortcut] npm module.
```

Comment:
I'm not convinced on recommending this (I can't find the last comment I made about it).  It's a cool trick but what it's doing behind the scenes is registering a global shortcut on focus and unregistering on blur.

This can lead to unknown side affects in other applications as lets say program X not based on Chromium has a global shortcut for "Control + Alt + Z", if you register a local shortcut through this module for the same combination then program X won't receive the global keypresses.  This isn't the only possible side-affect but an easy one to explain.

Whereas if you use mousetrap in a per-window style thing as mentioned below you don't have this problem.  Local shortcuts IMO shouldn't interfere with other global shortcuts and this module causes them to.  Perhaps it's something that should be discussed or we need to add an enhancement issue for adding first party local shortcuts based on the Accelerator class (I spiked this at one point but it seemed very resource intensive the way I did it).